### PR TITLE
added bookings#create, validates for booking model & routes modificat…

### PR DESCRIPTION
**Pull request by Tom-Firth**
Method create in bookingsController, pls check.
Validates for booking model => with only integer, defaut is not nil.. ;) 
& routes modification for have drone's id for booking.drone = params[:id]..

that's cool mates ?

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,2 +1,20 @@
 class BookingsController < ApplicationController
+
+  def create
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
+    @booking.drone = params[:id]
+
+    if @booking.save
+      redirect_to @booking
```
Comment (SabrinaNayeli) [inline app/controllers/bookings_controller.rb:17]: I don't understand your redirection, because there is no booking template...

**Review by ghost — COMMENTED** [review]
Good !

**Comment by AndyLukac** [comment]
looks good ;0

**Review by ghost — COMMENTED** [review]
Like it 👌🏻

**Pull request merged**